test/integration.sh

```diff
@@ -37,6 +37,18 @@ token=$(curl --request POST \
   --data client_secret=ZXhhbXBsZS1hcHAtc2VjcmV0 \
   --data scope="openid email" | sed 's/^{.*"id_token":[^"]*"\([^"]*\)".*}/\1/')
 
+(GUBER_MEMBERLIST_KNOWN_NODES=127.0.0.1:7946 gubernator-v0.8.4-0.20200617200142-07e238f8cd86) &
+
+
+echo "-------------------------------------------"
+echo "- Waiting for Gubernator to come up...  -"
+echo "-------------------------------------------"
+
+until curl --output /dev/null --silent --fail --insecure http://127.0.0.1:80/v1/HealthCheck; do
```

Comment:
There is bug in memberlist SD implementation of gubenator that restraints us to change the default port. Memberlist SD won't be used for production.

Reply:
Is there a way to block it, add some useful information to not forget about this PR comment?

Reply:
That comment is probably also really good to put into the code at this exact place. :blush: